src/tools/connection-info.ts

```
@@ -0,0 +1,154 @@
+        timezone: 'America/Los_Angeles'
+      },
+      connection: {
+        type: 'broadband',
+        asn: 12345,
+        organization: 'Example Internet Provider'
+      }
+    };
+
+    // Conditionally include location details
+    if (options.includeLocation === false && connectionData.location) {
+      delete connectionData.location.coordinates;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)